TIPS_and_FAQ.md

```diff
@@ -2,6 +2,47 @@
 
 ### As a user, how can I...
 
+#### Embedded NodeJS
+
+When a user's system suits the supported OSs (Linux, MacOS and Win32) and architectures (x86_64) the Embedded Node.js from Wild Web Developer will be used by WildWebDeveloper to run language Servers as well as for Node Debugger unless the `"org.eclipse.wildwebdeveloper.nodeJSLocation"` system property is set.
+
+Developers may use the NodeJS Embedder by adding the `org.eclipse.wildwebdeveloper.embedder.node.feature` into the dependencies, and calling the `getNodeJsLocation()` method of ``NodeJSManager` class in order to automatic install (if it's not yet installed) and obtain the embedded version of NodeJS in their products.
```

Comment:
I believe ``NodeJSManager` should be changed to `NodeJSManager` (remove one of the extra back ticks, as the rest of the text of the sentence is considered to be monospaced